src/gluonts/mx/block/regularization.py

```diff
@@ -81,7 +81,7 @@ def hybrid_forward(self, F, *states: List[Tensor]) -> Tensor:
                     state = F.stack(*state, axis=self._time_axis)
                 means.append(
                     self._alpha
-                    * state.__pow__(2).mean(
+                    * F.power(state, 2).mean(
```

Comment:
Do you need to use `F` here?

Reply:
`state` above can change type, I tried asserting that it's a `Tensor` (which supports `__pow__`) but that cannot be done apparently, so using `F.power` fixes it